piksi_tools/serial_link.py

```diff
@@ -275,7 +280,7 @@ def scallback(sbp_msg, **metadata):
     return scallback
 
 
-def run(args, link):
+def run(args, link, stop_function=None):
```

Comment:
If this is just `stop_function=lambda: None` you won’t need any of the `if stop_function is not None` checks.